src/test/regress/expected/multi_extension.out

```diff
@@ -543,9 +543,10 @@ SELECT * FROM print_extension_changes();
 -- Snapshot of state at 10.1-1
 ALTER EXTENSION citus UPDATE TO '10.1-1';
 SELECT * FROM print_extension_changes();
- previous_object | current_object
+ previous_object |              current_object
```

Comment:
why do we see this in `'10.1-1';`?

Reply:
we already talked in teams but sharing the link for reference
https://github.com/citusdata/citus/pull/4747/commits/dcc020760519edda38753d176ff3c9ac718640cf